.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Release Build
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v2
+      with:
+        java-version: 11
+        distribution: 'temurin'
+    - name: Setup Gradle
+      uses: gradle/gradle-build-action@v2
+    - name: Install gradle wrapper
+      run: gradle wrapper
+    - name: Execute Gradle build
+      run: ./gradlew build
+
+    - name: Build APK
+      run: bash ./gradlew assembleRelease
```

Comment:
The can be shortened, and also the correct command is: `./gradlew assembleProductionRelease`